### PR TITLE
Add Cocoapods support & removed deprecated React.Proptypes reference

### PR DESCRIPTION
**Pull request by RobertBiehl**
I added a *.podspec for this project so people who are using Cocoapods with React can integrate it more easily.

### CODE REVIEW

**Comment by RobertBiehl** [comment]
@umhan35 Can you please merge this? It's a great help for Cocoapods users.

**Comment by bkvirendra** [comment]
@RobertBiehl maybe you want to update the `package.json` file to include the `ReactNativeSearchBar.podspec` in the package on getting published.

**Comment by RobertBiehl** [comment]
@bkvirendra Do you mean updating the app version? The package.json does not seem to reference any files, or did I miss something?

File: ReactNativeSearchBar.podspec [hunk 1]
```diff
@@ -0,0 +1,17 @@
+require "json"
+package = JSON.parse(File.read(File.join(__dir__, "package.json")))
+
+Pod::Spec.new do |s|
+  s.name          = "ReactNativeSearchBar"
+  s.version       = package["version"]
+  s.source_files  = "**/*.{h,m}"
```
Comment (derrh) [inline ReactNativeSearchBar.podspec]: This pattern matches the sample code. Please update it to `"RNSearchBar*.{h,m}"` or perhaps `"RNSearchBar.{h,m}", "RNSearchBarManager.{h,m}"`

**Comment by iRoachie** [comment]
Hey @RobertBiehl new maintainer here. We recently released [v3.1.0](https://github.com/umhan35/react-native-search-bar/releases/tag/v3.1.0) so now your changes are out of sync with latest. Can you rebase your branch against the `next` branch?

**Comment by iRoachie** [comment]
I merged your commit @RobertBiehl in https://github.com/umhan35/react-native-search-bar/commit/c2499e8894b83f313dbf0bd2b3c36b53efed4bd6. Will be available in next release. Thanks again.

**Pull request closed without merging**